### PR TITLE
Hotfix/ad layers size mappings format fix

### PR DESCRIPTION
**Pull request by jomurgel**
- addresses issues where size mappings didn't return values by name.
- reverse js update.

### CODE REVIEW

File: inc/ad-servers/class-ad-layers-dfp.php [hunk 1]
```diff
@@ -534,8 +534,14 @@ private function ad_unit_js( $ad_layer ) {
...
 
+					// If we have no defaults, assume to use all of the sizes for this breakpoint.
+					if ( empty( $this->default_by_unit[ $ad_unit['code'] ] ) ) {
+						$this->default_by_unit[ $ad_unit['code'] ] = $sizes;
+					}
+
 					// Generate the mapping JS and store it with the unit.
-					$unit_key = $this->sanitize_key( $ad_unit['code'] );
+					// TODO: streamline targeting so mappings and units are 1:1.
+					$unit_key = $ad_unit['code'];
```
Comment (mogmarsh) [inline inc/ad-servers/class-ad-layers-dfp.php:544]: is removing the call to sanitize_key going to cause issues? We may end up with some weird characters in the key, and it removes a call to a filter that people may be using.

File: entries/ad-layers-dfp/index.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ import './style.scss';
         args.targeting = dfpAdDetails[args.format].targeting;
       }
       if (!args.sizeMapping) {
-        if (dfpBuiltMappings && dfpBuiltMappings[args.format]) {
+        if (dfpBuiltMappings) {
           args.sizeMapping = dfpBuiltMappings[args.format];
```
Comment (mogmarsh) [inline entries/ad-layers-dfp/index.js:104]: do we need a failover here since you aren't checking if the array value is set? Or will it always be set?

**Review by mogmarsh — APPROVED** [review]
I had a couple questions, but I don't think anything is blocking. 👍

File: inc/ad-servers/class-ad-layers-dfp.php (hunk 1 above)
Comment (jomurgel) [reply inc/ad-servers/class-ad-layers-dfp.php:544]: Yes, I never updated that to use `sanitize_title_with_dashes` or remove the todo. Thanks for the reminder.

File: entries/ad-layers-dfp/index.js (hunk 2 above)
Comment (jomurgel) [reply entries/ad-layers-dfp/index.js:104]: reverted this change.

**Pull request merged**